packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -47,26 +47,12 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
 # start package.json devDependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(babel-cli) >= 6.10.1
-BuildRequires: %{?scl_prefix}npm(babel-cli) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-core) >= 6.26.3
-BuildRequires: %{?scl_prefix}npm(babel-core) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-loader) >= 7.1.1
-BuildRequires: %{?scl_prefix}npm(babel-loader) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-module-resolver) >= 3.2.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-module-resolver) < 4.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-syntax-dynamic-import) >= 6.18.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-syntax-dynamic-import) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-class-properties) >= 6.24.1
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-class-properties) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-assign) >= 6.8.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-assign) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-rest-spread) >= 6.8.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-rest-spread) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-env) >= 1.7.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-env) < 2.0.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-react) >= 6.5.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-react) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.3.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(babel-loader) >= 8.0.0
+BuildRequires: %{?scl_prefix}npm(babel-loader) < 9.0.0
 BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
 BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
 BuildRequires: %{?scl_prefix}npm(jed) >= 1.1.1
```

Comment:
Not a blocker to merge since this is correct in that it reflects what's in the package, but I'm surprised these weren't dropped since I believe a lot of this should be provided by vendor. @sharvit can you confirm that the plugin should actually import patternfly (and others) from the vendor package or rectify my expectations?

Reply:
pattenfly-react should be imported from the `vendor`, see packages supply by the vendor:
https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L28-L85

Reply:
I suppose that requires code change in tasks which was not done in the vendor 3 PR, probably work for 1.0.1, but this PR is now on hold there's a chance to fix that before new 1.0.x release

Reply:
There is confusion because there are 2 packages
- patternfly - bring css styles - loaded as a gem (patternfly-sass)
- patternfly-react - bring react components - supplied by the vendor

**foreman-tasks have `patternfly` in the package.json so it can be used in the storybook.**
Now, when we migrated storybook to @theforeman/stories, we can remove `patternfly` from foreman-tasks.